### PR TITLE
Update spelling section 

### PR DESCRIPTION
**Pull request by nastasha-solomon**
Rewrote and expanded on existing spelling guidelines for UI copy. Some of the changes I made:
- Defined American English
- Outlined core differences in the way certain suffixes are spelled
- Provided examples of how words are spelled in American English vs British English

### CODE REVIEW

File: website/docs/03_content/spelling.mdx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+description: Guidelines for using consistent spelling and terminology in the UI.
+---
+
+Use American English when writing UI copy. American English is a version of the English language used in the United States. It's sometimes called United States English or U.S. English.
+
+Certain words are spelled differently in American English and British English. You'll find some of these key spelling differences in the following sections.  
+
+## Verbs that end with 'ize' or 'yze'
+
+In American English, verbs that end with 'ize' will end with 'ize' or 'ise' in British English. Similary, verbs that end with 'yze' in American English will end with 'yse' in British English.
```
Comment (mdbirnstiehl) [inline website/docs/03_content/spelling.mdx]:
```suggestion
In American English, verbs that end with 'ize' usually end with 'ise' in British English. Similarly, verbs that end with 'yze' in American English usually end with 'yse' in British English. 
```
I think we can maybe remove the -ize examples from British English and focus on the differences with the -ise spellings.

File: website/docs/03_content/spelling.mdx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+## Verbs that end with 'ize' or 'yze'
+
+In American English, verbs that end with 'ize' will end with 'ize' or 'ise' in British English. Similary, verbs that end with 'yze' in American English will end with 'yse' in British English. 
+
+| American English | British English | 
+| ------------- |:-------------:|
+| organize | organise _or_ organize | 
+| authorize | authorise _or_ authorize|  
+| analyze | analyse _or_ analyze |
```
Comment (mdbirnstiehl) [inline website/docs/03_content/spelling.mdx]:
```suggestion
| organize | organise | 
| authorize | authorise |  
| analyze | analyse | 
```

File: website/docs/03_content/spelling.mdx [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+| American English | British English | 
+| ------------- |:-------------:|
+| license | licence | 
+| defense | defence |  
+| pretense | pretence | 
+ 
+## Nouns that end with 'og' or 'ogue' 
+
+In American English, nouns that end with 'og' or 'ogue' usually end with 'ogue' in British English.
```
Comment (mdbirnstiehl) [inline website/docs/03_content/spelling.mdx]:
```suggestion
In American English, nouns that end with 'og' usually end with 'ogue' in British English.
```
To focus on the different spellings here.

**Review by mdbirnstiehl — APPROVED** [review]
Just a couple of suggestions to accept of deny at your discretion 🚀

File: website/docs/03_content/spelling.mdx [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+In American English, nouns that end with 'ense' usually end with 'ence' in British English.
+
+| American English | British English | 
+| ------------- |:-------------:|
+| license | licence | 
+| defense | defence |  
+| pretense | pretence | 
+ 
+## Nouns that end with 'og' or 'ogue'
```
Comment (nastasha-solomon) [inline website/docs/03_content/spelling.mdx:41]: Just noticed that there's a small typo here I'll need to fix. It should be: 
Nouns that end with `og`

**Pull request merged**